presto-hive/src/main/java/io/prestosql/plugin/hive/s3/S3FileSystemType.java

```diff
@@ -17,4 +17,5 @@
 {
     PRESTO,
     EMRFS,
+    CUSTOM,
```

Comment:
@kokosing proposed "HADOOP"

I'd propose "HADOOP_DEFAULT".

Reply:
> * `HADOOP_DEFAULT` may remind users the other Hadoop property `fs.defaultFS`

I see your point.
To me it stands for "Hadoop's default".
I prefer to have this "default" there, because "Hadoop" is not a filesystem implementation.

(In fact, we internally already use HADOOP_DEFAULT with exactly same meaning (for some service other than file system))

Reply:
Also it could be named `NONE` because we don't set anything in Presto and one may not load any resource files.

Reply:
I don't like NONE, as we may end up having s3 fs impl.
@kokosing WDYT about what I suggested?

Reply:
Sounds good to me.